### PR TITLE
Introduce support for TemSen with fallback and version checks

### PR DESCRIPTION
**Pull request by cmroche**
Introduces support for TemSen with a fallback for unrecognized cases or None value.

See #26 for the conversation history.

CLOSES: #13

### CODE REVIEW

File: greeclimate/device.py [hunk 1]
```diff
@@ -290,7 +290,8 @@ def temperature_units(self, value: int):
 
     @property
     def current_temperature(self) -> int:
-        return self.get_property(Props.TEMP_SENSOR)
+        prop = self.get_property(Props.TEMP_SENSOR)
+        return prop or self.target_temperature
```
Comment (xuefer) [inline greeclimate/device.py]: consider 0C
return prop if prop is not None else self.target_temperature

**Review by xuefer — CHANGES_REQUESTED** [review]

File: greeclimate/device.py [hunk 2]
```diff
@@ -291,7 +307,14 @@ def temperature_units(self, value: int):
     @property
     def current_temperature(self) -> int:
         prop = self.get_property(Props.TEMP_SENSOR)
-        return prop or self.target_temperature
+        if prop:
```
Comment (xuefer) [inline greeclimate/device.py:310]: consider 0C
please use "if prop is not None:"

**Review by xuefer — CHANGES_REQUESTED** [review]

File: greeclimate/device.py (hunk 2 above)
Comment (cmroche) [reply greeclimate/device.py]: Done, added a test for this case too.

**Comment by cmroche** [comment]
@xuefer I've integrated and testing this change into a branch of the HomeAssistant component now. Unfortunately, HA is very slow to get through MR's and I have several pending, it could be months before this feature is included in an official release, but if you would like to start using it you can easily install as a custom component and it should seamlessly override the component setup in HA.

You can setup the custom component with this command:

`wget -O - https://raw.githubusercontent.com/cmroche/HA-GreeComponentInstaller/master/install-test.sh | bash`

And you can inspect the script here: https://github.com/cmroche/HA-GreeComponentInstaller

**Pull request merged**